pyflagser/flagio.py

```diff
@@ -25,7 +25,14 @@ def loadflag(fname, fmt='csr', dtype=None):
     -------
     flag_matrix : matrix of format `fmt`
         Matrix representation of a directed/undirected weighted/unweighted
-        graph. Diagonal elements are vertex weights.
+        graph. Diagonal elements are vertex weights. The way zero values are
+        handled depends on the format of the matrix. If the matrix is a dense
```

Comment:
Would it be more explicit and preferrable to say "The way zero values are handled depends on `fmt`", since this is how the user would control the behaviour?

Reply:
Then one would say e.g. "If `fmt` it `'dense'`", etc.